study.md

```diff
@@ -44,21 +48,30 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- "View" is essentially the portion HTML is making up what the users see that they might refer to as a new page or screen.
+
+https://johnpapa.net/pageinspa/-->
 ```
```

Comment:
Awesome! View is what the user is seeing on the page. View state is referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state. The front end router is what manages view-state by responding to user inputs, and directing them to the appropriate view!
Take a look at this repo https://github.com/ga-wdi-boston/spa-study/blob/master/study.md